### PR TITLE
feat: add test setup based on jest and testing-library

### PR DESCRIPTION
**Pull request by marcosvega91**
I think that I have added what we need for testing.

In `setupTests.js` I have assigned to window object the export from testing library.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,9 +8,12 @@
     "build": "parcel build index.html --no-cache",
     "lint": "eslint 'src/**/*.js' --quiet --fix",
     "clean": "rimraf ./dist ./.cache",
-    "ci:lint": "eslint 'src/**/*.js' -c ./.eslintrc.js"
+    "ci:lint": "eslint 'src/**/*.js' -c ./.eslintrc.js",
+    "test": "jest",
+    "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@testing-library/dom": "^7.5.7",
```
Comment (smeijer) [inline package.json:16]: This should be added to dev dependencies, as testing-library is being included from a cdn at the moment.

See comment here for why it is that way:

https://github.com/smeijer/testing-playground/blob/6420421a24e8792f30fccc8482d04d732125d30a/src/parser.js#L5:L11

Comment (marcosvega91) [reply package.json:16]: I have added here for a future implementation. I move it in dev dep

**Comment by smeijer** [comment]
Thanks. Looking great!

One dependency should be moved though. 

Also what do you think? Should we add one single "app doesn't throw on render" test just to verify that the setup is working?

File: package.json (hunk 1 above)
Comment (smeijer) [reply package.json:16]: Yeah, I expected that. But I was afraid that it will end up in the production bundle. Thanks.

**Comment by smeijer** [comment]
And last one, do we need to add  https://github.com/testing-library/user-event?

**Comment by marcosvega91** [comment]
I have added the other things that we need. I also create a test, as you suggest, for App.js :)

**Comment by smeijer** [comment]
Awesome! Thanks.

**Pull request merged**